apyros/metalog.py

```diff
@@ -57,7 +71,11 @@ def getLog( self, prefix ):
                 ret = line.split()[1].strip()
```

Comment:
well, I change that, but it may fail in cases like "c:\strange\path" or 12:30:57 ...